muswarmadmin/main.py

```diff
@@ -176,7 +176,7 @@ def docker(self):
         result = await self.sparql.query("""
             ASK FROM {{graph}} WHERE { ?s mu:uuid {{}} }
             """, escape_string(resource_id))
-        return False if result is None else result['boolean'] 
```

Comment:
I don't know why I accepted this in the first place but the result of a ASK query is a boolean and not an empty body. Please revert it and fix the problem in the delta service where it should be.